WordPress/gutenberg
# Block manager: Display a 'Reset' button when blocks are hidden to quickly enable all

https://playground.wordpress.net/?gutenberg-pr=51200
Related to https://github.com/WordPress/gutenberg/issues/50015
## What?


It adds a "Reset" button in order to easily show all the hidden blocks in the Preferences section.

## Why?


Right now, when there are many hidden blocks, the user must:
- Enable them again one by one
- Enable them in bulk by clicking on the category of each block

The first one is time-consuming, and the second one is a bit tricky, as it's not particularly obvious to the end user.

Adding a new button specific to that use case will make things easier for the end user.

## Testing Instructions





1. Edit a page
2. Navigate to "Preferences" in the top right corner
3. Make sure that you uncheck some blocks, after visiting "Panels" menu in the sidebar.
4. Ensure that you see an "Reset" button.
5. Ensure that when you press it, all blocks are checked again.

### Testing Instructions for Keyboard


## Screenshots or screencast 

<img width="559" alt="image" src="https://github.com/WordPress/gutenberg/assets/36432/f903e90c-910d-438a-906c-0ac478ae4841">
